site/portfolio.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+
+<html>
+	<head>
+		<title>Portfolio</title>
+
+		<link rel="stylesheet" href="style.css">
+		<link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Lato:400,900' rel='stylesheet' type='text/css'>
+	</head>
+
+	<body>
+		<ul class="top_nav">
+			<li class="top_nav_item"><a href="about.html" class="li a">About me</a></li>
+			<li class="top_nav_item"><a href="portfolio.html" class="active">Portfolio</a></li>
+			<li class="top_nav_item"><a href="another_page.html" class="li a">Another Page</a></li>
+		</ul>
+
+		<div class="name_area">
+			<h1>I LOVE WHAT I DO.</h1>
+		</div>
+
+		<p class="portfolio_intro">I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer!</p>
+
+		<hr>
+
+		<div class="experience">
+			<h2>EXPERIENCE</h2>
+
+			<table>
```

Comment:
I love that you are experimenting with other HTML tags that we have not covered yet in class.  This is a great implementation of a table from a code perspective.  However, it has been our best practice for a while that developers only use tables for tabular data (think, Excel spreadsheet worthy) and not for organizing and laying out content like images and text.

There are other ways to achieve this look of having your image and text side-by-side using a property called "float".  We will dive more into this in our next class on the 29th.